src/pages/workspace/WorkspaceJoinUserPage.tsx

```diff
@@ -39,10 +39,10 @@ function WorkspaceJoinUserPage({route, policy}: WorkspaceJoinUserPageProps) {
     }, []);
 
     useEffect(() => {
-        if (!policy || isUnmounted.current || isJoinLinkUsed) {
+        if (isUnmounted.current || isJoinLinkUsed) {
             return;
         }
-        if (!isEmptyObject(policy)) {
+        if (!isEmptyObject(policy) && !policy?.isJoinRequestPending) {
```

Comment:
@nkdengineer Why change here?

Reply:
After we request to join the workspace, policy will not be empty. So if we click the join link again, we will go to initial page. `!policy?.isJoinRequestPending` is added to prevent it. `!isEmptyObject(policy) && !policy?.isJoinRequestPending` is the same with `isPolicyMember` check.